config/v1/feature_gates_test.go

```diff
@@ -0,0 +1,58 @@
+package v1
+
+import (
+	"fmt"
+	"testing"
+
+	utilerrors "k8s.io/apimachinery/pkg/util/errors"
+)
+
+func validateFeatureGateDescription(in FeatureGateDescription) error {
+	errs := []error{}
+
+	if len(in.FeatureGateAttributes.Name) == 0 {
```

Comment:
Any interest in regex validating these fields?

Reply:
> Any interest in regex validating these fields?


happy to see it as a followup.